src/replica/DirectorIndexJob.cc

```diff
@@ -388,18 +386,19 @@ void DirectorIndexJob::_loadDataIntoTable() {
         if (request == nullptr) break;
 
         // Load request's data into the destination table.
+        bool const localFile = true;
         try {
             string const query = g.loadDataInfile(
                     request->responseData().fileName, directorIndexTableName(database(), directorTable()),
                     controller()->serviceProvider()->config()->get<string>("worker", "ingest-charset-name"),
-                    localFile());
+                    localFile);
```

Comment:
Would it be better to remove the parameter from `loadDataInfile`?

Reply:
This can't be done as this option is still used by the worker ingest service for loading chunk contributions. In this case, both worker Ingest service and worker MySQL share the folder as they are guaranteed to be run within the same Kubernetes pod (and on the same host in the Docker-based environment). The `local=0` mode provides a noticeable speedup in the performance of the bulk ingest operations. We just can't do this for constructing the "director" index.